src/connection/hive/HivePublishUI.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button } from '@neo4j-ndl/react';
+import SaveToHiveModal from '../../extensions/hive/components/SaveToHiveModal';
+import { CloudArrowUpIconOutline } from '@neo4j-ndl/react/icons';
+
+export function getHivePublishUIButton(_params: any): any {
```

Comment:
Perhaps this file should live in `src/extensions/hive` instead?

Reply:
sure, I can move it

Reply:
moved